### PR TITLE
fix(seed): Missing/duplicate info

### PR DESCRIPTION
**Pull request by BerkeleyTrue**

### CODE REVIEW

File: seed/challenges/02-javascript-algorithms-and-data-structures/javascript-algorithms-and-data-structures-certificate.json [hunk 1]
```diff
@@ -1,232 +0,0 @@
-{
```
Comment (BerkeleyTrue) [inline seed/challenges/02-javascript-algorithms-and-data-structures/javascript-algorithms-and-data-structures-certificate.json]: duplicate file

File: seed/challenges/02-javascript-algorithms-and-data-structures/es6.json [hunk 2]
```diff
@@ -760,72 +760,6 @@
       "translations": {}
     },
     {
-      "id": "587d7b8b367417b2b2512b51",
```
Comment (BerkeleyTrue) [inline seed/challenges/02-javascript-algorithms-and-data-structures/es6.json]: duplicate challenge

File: seed/challenges/02-javascript-algorithms-and-data-structures/basic-data-structures.json [hunk 3]
```diff
@@ -1,52 +1,10 @@
 {
-  "name": "Debugging",
+  "name": "Basic Data Structures",
   "order": 4,
   "time": "1 hour",
   "helpRoom": "Help",
   "challenges": [
     {
-      "id": "587d78b2367417b2b2512b0d",
```
Comment (BerkeleyTrue) [inline seed/challenges/02-javascript-algorithms-and-data-structures/basic-data-structures.json]: duplicate challenge

File: seed/challenges/02-javascript-algorithms-and-data-structures/basic-data-structures.json [hunk 4]
```diff
@@ -1,52 +1,10 @@
 {
-  "name": "Debugging",
```
Comment (BerkeleyTrue) [inline seed/challenges/02-javascript-algorithms-and-data-structures/basic-data-structures.json]: Wrong name

File: seed/challenges/01-responsive-web-design/basic-css.json [hunk 5]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Basic HTML and HTML5",
```
Comment (BerkeleyTrue) [inline seed/challenges/01-responsive-web-design/basic-css.json]: Wrong name

**Comment by QuincyLarson** [comment]
Thanks for catching these. LGTM!

**Pull request merged**